notebooks/278-mobilevlm-language-assistant/278-mobilevlm-language-assistant.ipynb

```diff
@@ -0,0 +1,783 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    if not Path(f"{MODELS_DIR}/stage1.xml").exists():
</pre><p>Could you please add a prefix to the path containing an indication of the compression mode, int4 or int8? Otherwise, you need to delete the model to use a different type of compression mode.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1627/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>